staging/src/k8s.io/client-go/README.md

```diff
@@ -1,9 +1,28 @@
-This _staging/src/k8s.io/client-go directory is the staging area of the client repo. It contains a versioned client, tools built around the client like the reflector, and all the client dependencies. The content will be periodically published to k8s.io/client-go repo.
-The staged content is copied from the main repo, i.e., k8s.io/kubernetes, with directory rearrangement and necessary rewritings. To sync the content with the latest code in your local k8s.io/kubernetes, you need to run `godep restore` in k8s root directory, then run _staging/src/k8s.io/client-go/copy.sh.
-vendor/k8s.io/client-go is a symlink pointing to this staging area, so to use the packages in the staging area, you can import it as "vendor/client-go/<package-name>", as if the client were vendored. The client will be vendored from k8s.io/client-go for real after the test matrix is converted to vendor k8s components.
+# client-go
 
+Go clients for talking to a [kubernetes](http://kubernetes.io/) cluster.
 
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/_staging/src/k8s.io/client-go/README.md?pixel)]()
+### What's included
 
+* The `kuberentes` package contains the clientset to access Kubernetes API.
+* The `discovery` package is used to discovery API supported on the Kubernetes API server.
+* The `dynamic` package contains a dynamic client that can handle API objects without knowing the scheme.
+* The `transport` package is used to set up auth and start a connection.
+* The `tools/cache` package is useful for writing controllers.
 
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/staging/src/k8s.io/client-go/README.md?pixel)]()
+### Releases
+
+Each top-level folder (e.g., 1.4) contains a release of clients and their dependencies.
+
+client-go has the same release cycle as the Kubernetes main repository. For example, in the 1.4 release cycle, the contents in `1.4/` folder are subjected to changes. The `1.4/` folder will be stable once the 1.4 release is final, and new changes will go into the `1.5/` folder.
```

Comment:
Add: We will make great efforts to not change the public interface of a version of the client once that version has been released. We may change the interface between versions. Old versions of the client will be retained for two release cycles.